.github/workflows/macos_intel_install.yml

```diff
@@ -92,19 +92,19 @@ jobs:
         run: |
           source ~/openquake/bin/activate
           cd ~/work/oq-engine/oq-engine
```

Comment:
Please add a comment about why we are using `-v 2` (I guess it is the verbosity level).

Reply:
Done